bin/setversion

```diff
@@ -32,7 +32,8 @@ if (empty($version)) {
 $versioner = new \NamelessCoder\TYPO3RepositoryClient\Versioner();
 
 try {
-    $result = $versioner->write($directory, $version, $stability);
+    $version = str_replace('v', '', $version);
```

Comment:
I'm thinking this part is redundant since it is also done in Versioner. I'd prefer to do it in Versioner only, since that class may theoretically be used separately from the CLI command.

Reply:
ok then the echo will have the v prefix as well below.